src/hooks/use-stores.test.ts

```diff
@@ -0,0 +1,115 @@
+import { ProblemModel } from "../models/curriculum/problem";
+import { AppConfigModel } from "../models/stores/app-config-model";
+import { ClassModel } from "../models/stores/class";
+import { DemoClassModel, DemoModel } from "../models/stores/demo";
+import { DocumentsModel } from "../models/stores/documents";
+import { GroupsModel } from "../models/stores/groups";
+import { SelectionStoreModel } from "../models/stores/selection";
+import { UIModel } from "../models/stores/ui";
+import { UserModel } from "../models/stores/user";
+import { LearningLogWorkspace, ProblemWorkspace, WorkspaceModel } from "../models/stores/workspace";
+import {
+  useAppConfigStore, useAppMode, useClassStore, useDemoStore, useGroupsStore, useNetworkDocumentKey, useProblemPath,
+  useProblemPathWithFacet, useProblemStore, useSharedSelectionStore, useTypeOfTileInDocumentOrCurriculum,
+  useUIStore, useUserStore
+} from "./use-stores";
+
+jest.mock("@concord-consortium/slate-editor", () => ({}));
+
+var mockUseContext = jest.fn();
+jest.mock("react", () => ({
+  ...jest.requireActual("react"),
+  useContext: () => mockUseContext()
+}));
+
+describe("useStores", () => {
```

Comment:
My assumption is that some of these tests ("useStores", "simple store hooks") are actually adding test coverage for existing code, as opposed to testing the new work (i.e., "useTypeOfTileInDocumentOrCurriculum"). Which is fine, BTW, in case that's not clear. But I'm going to just grade them with a 'LGTM', as I don't feel that I have enough familiarity with the existing code to do a more accurate review.

Reply:
True -- the proximate reason for adding the `use-stores.test.ts` is that I wanted to add tests for the new code and there wasn't a good place to do so without introducing the new file. That said, the lack of this test file also indicated that there was other code in `use-stores.ts` that was similarly untested and so I took a few extra minutes to flesh out tests for some preexisting code while I was in the neighborhood.